content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -15,14 +15,19 @@ is_beta: true
 The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
 </div>
 
-### Oracle versions supported
+Supported:
+- versions: 19c and 21c
+- deployment configurations: self-managed, RDS, RAC, Exadata
+- architecture: multi-tenant
 
-|  | Self-hosted | RDS |
-|--|-------------|-----| 
-| Oracle 19c | {{< X >}} | {{< X >}} |
-| Oracle 21c | {{< X >}} | {{< X >}} |
+Not supported deployment configurations, components and features: Oracle Autonomous Databae, ASM, Data Guard.
```

Comment:
Databae --> Database (typo)

Do we want to specify that we don't support legacy versions here?

Reply:
Added now